data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/schema/GetSyncCommitteeRewards.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title" : "GetSyncCommitteeRewards",
+  "type" : "object",
+  "required" : [ "execution_optimistic", "finalized", "data" ],
+  "properties" : {
+    "execution_optimistic" : {
+      "type" : "boolean"
+    },
+    "finalized" : {
+      "type" : "boolean"
+    },
+    "data" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "required" : [ "validator_index", "reward" ],
+        "properties" : {
+          "validator_index" : {
+            "type" : "string",
+            "description" : "integer string",
+            "example" : "1",
+            "format" : "integer"
+          },
+          "reward" : {
+            "type" : "string",
+            "description" : "unsigned 64 bit integer",
+            "example" : "1",
+            "format" : "uint64"
+          }
```

Comment:
we need to see negatives in output, i believe this is a hint that it's not currently going to allow that...

Reply:
It never will be negative. The calculations min at 0.